stack/__main__.py

```diff
@@ -325,6 +369,7 @@ def add_bucket_permissions(
         roles=[gcp.cloudidentity.GroupMembershipRoleArgs(name='MEMBER')],
     )
 
+    # TODO(@lgruen): remove this once secrets are used for checking memberships.
```

Comment:
This one too

Reply:
Not yet, we need to update the analysis-runner / web server first.